packages/drafts-realm/ai-app-generator.gts

```diff
@@ -0,0 +1,452 @@
+            <section>
+              <header class='db-content-header'>
+                <h2 class='prompt-title'>Generate an App</h2>
+              </header>
+              <CardContainer @displayBoundaries={{true}} class='prompt'>
+                <div class='prompt-options'>
+                  <button class='prompt-option active'>Start From Scratch</button>
+                  <button disabled class='prompt-option'>Remix an Existing App</button>
+                </div>
+                <section class='prd-editor'>
+                  <FieldContainer @label='I want to make a'>
+                    <BoxelInput
+                      @value={{this.prompt.appType}}
+                      @onInput={{fn this.setPrompt 'appType'}}
+                    />
+                  </FieldContainer>
+                  <FieldContainer @label='Tailored for'>
+                    <BoxelInput
+                      @value={{this.prompt.domain}}
+                      @onInput={{fn this.setPrompt 'domain'}}
+                    />
+                  </FieldContainer>
+                  <FieldContainer
+                    class='features'
+                    @label='That has these features'
+                  >
+                    <BoxelInput
+                      @value={{this.prompt.customRequirements}}
+                      @onInput={{fn this.setPrompt 'customRequirements'}}
+                    />
+                  </FieldContainer>
+                  <Button
+                    class='generate-button'
+                    @kind='primary-dark'
+                    {{on 'click' this.generatePrd}}
+                  >
+                    <span class='button-logo' />
+                    Let's Get Started
+                  </Button>
+                </section>
+              </CardContainer>
+            </section>
+            <aside>
+              <header class='db-content-header'>
+                <h3 class='sample-apps-title'>Browse Sample Apps</h3>
+              </header>
+              <ul class='sample-apps-list'>
+                {{#each this.instances key='id' as |card|}}
+                  <li
+                    class='sample-apps-list-item'
+                    {{@context.cardComponentModifier
+                      card=card
+                      format='data'
+                      fieldType=undefined
+                      fieldName=undefined
+                    }}
+                  >
+                    {{#let (getComponent card) as |Card|}}
+                      <Card />
+                    {{/let}}
+                  </li>
+                {{/each}}
+              </ul>
+            </aside>
+          </div>
+        {{/if}}
+      </div>
+    </section>
+
+    <style>
+      h1,
+      h2,
+      h3,
+      h4,
+      h5,
+      h6,
+      p {
+        margin-top: 0;
+        margin-bottom: var(--boxel-sp);
+      }
+      ul {
+        list-style-type: none;
+        margin: 0;
+        padding: 0;
+      }
```

Comment:
This seems very generic. Is it for styling markdown output? If so, does this CSS belong in the Markdown field?

Reply:
no, in many places where `ul` element is used. when tabbed-header is refactored, there might be less need of this.